.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: scala
 scala:
-  - 2.10.6
```

Comment:
Dropping 2.10 support is ok for me but we should mention it in the announcement of 2.5.0 release.